projects/habitat_uncertainty/config/nav_obj_rl_skill.yaml

```diff
@@ -0,0 +1,166 @@
+# @package _global_
```

Comment:
Why create a separate config or each skill ? Is this to get around the slurm requeing issue ?

I think maybe creating specific configs for each individual experiment will eventually lead to too many configs (although I know I suggested this).  A better way is to use a single base config and then override parameters through command line in the slurm batch script itself and then you can have multiple slurm batch scripts which will correctly resubmit your jobs as the command line overrides remain same

Reply:
Yes, it was originally done for slurm requeing.
Specific parameters targeted are

1. RGB camera height and width
2. Include or remove RGB and depth input from resizing parameter
3. Change policy name

I'll try to shift to make changes in slurm batch script

Reply:
Yeah, I think you should move to this framework of maintaining separate slurm batch files instead of maintain too many config files
Look at how the OVON code [overrides with commandline parameters](https://github.com/naokiyokoyama/ovon/blob/main/scripts/train/1-ovon-ver.sh#L35-L49) but uses a single base config